week-4/B-more-complex-objects/README.md

```diff
@@ -0,0 +1,5 @@
+### Goals
+
+- to continue solidifying understanding of objects
+- solve more complex exercises with objects
+- show how even more complex types can be assigned to object properties: arrays and even other objects

```

Comment:
Real world examples could include:
array of objects: search results, blog posts, social feed
object with array: person.interests

Reply:
cool, good examples!

Reply:
Also I suggest a section where we explicitly explain when to use Arrays vs. Objects ... this is an area that was never clear for the students in the last round. This article might be a good anology https://medium.freecodecamp.org/javascript-arrays-and-objects-are-just-like-books-and-newspapers-6e1cbd8a1746